README.md

```diff
@@ -31,6 +35,23 @@ The following environment variables are required for setting configuration and k
 - `VAULT_ADDR` environment variable (example: `http://127.0.0.1:8200`)
 - `CONSUL_ADDR_HTTP` environment variable (example: `http://127.0.0.1:8500`)
 
+The following may be optionally set in `~/.hashi-helper-config.yml.pgp` / `$HASHIHELPER_CONFIG_FILE`, a Keybase PGP-encrypted YAML file containing details about Vault server configurations and preferences for handling encrypted files:
+```yml
+---
+vault_profile_name_1:
+  server: http://<ip>:8200
+  token: <your token>
+vault_profile_name_2:
+  server: http://<ip>:8200
+  token: <your token>
+...
+encrypted-file-config:
+  keybase-team-name: < your keybase team name >
+  editor: < your favorite editor > # defaults to $EDITOR, this, then pico
+...
+```
+Note that if `--no-keybase-team` global flag is set, `keybase-team-name` will be ignored and only the local Keybase PGP profile will be used.
```

Comment:
if `keybase-team-name` is not set in yaml, will that be the same as `--no-keybase-team`?